front/src/config/i18n/fr.json

```diff
@@ -917,7 +919,11 @@
           "friday": "Vendredi",
           "saturday": "Samedi",
           "sunday": "Dimanche"
-        }
+        },
+        "house": "Maison"
+      },
+      "warning": {
+        "houseWithoutCoordinate": "Votre maison n'a pas de coordonnées renseignées. Ce déclencheur ne peut fonctionner sans cette donnée. Veuillez-vous rendre dans Paramètes/Maisons et cliquer sur la carte pour en associer une."
```

Comment:
1 mistake in "Paramètes" (without 'R') + end of the sentence is hard to understand.

I would change this to: 

"Veuillez-vous rendre dans Paramètes/Maisons et cliquer sur la carte pour en associer une." => "Veuillez-vous rendre dans Paramètres/Maisons et cliquer sur la carte pour définir les coordonnées de cette maison."